lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_send_app_size_metrics.rb

```diff
@@ -0,0 +1,165 @@
+
+          See https://github.com/Automattic/apps-metrics for the API contract expected by the Metrics server you are expected to send those metrics to.
+
+          Tip: If you provide a `file://` URL for the `api_url`, the action will write the payload on disk at the specified path instead of sending
+          the data to a endpoint over network. This can be useful e.g. to inspect the payload and debug it, or to store the metrics data as CI artefacts.
+        DETAILS
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :api_url,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_URL',
+            description: 'The endpoint API URL to publish metrics to. (Note: you can also point to a `file://` URL to write the payload to a file instead)',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :api_token,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_TOKEN',
+            description: 'The bearer token to call the API. Required, unless `api_url` is a `file://` URL',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :use_gzip_content_encoding,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_USE_GZIP_CONTENT_ENCODING',
+            description: 'Specify that we should use `Content-Encoding: gzip` and gzip the body when sending the request',
+            type: FastlaneCore::Boolean,
+            default_value: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_name,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_NAME',
+            description: 'The name of the app for which we are publishing metrics, to help filter by app in the dashboard',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_version,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_VERSION',
+            description: 'The version of the app for which we are publishing metrics, to help filter by version in the dashboard',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :build_type,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_BUILD_TYPE',
+            description: 'The build configuration for which we are publishing metrics, to help filter by build config in the dashboard. E.g. `Debug`, `Release`',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :source,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_SOURCE',
+            description: 'The type of event at the origin of that build, to help filter data in the dashboard. E.g. `pr`, `beta`, `final-release`',
```

Comment:
Nitpick: There might use cases other than a dashboard where we want to filter or group by event name:

```suggestion
            description: 'The type of event at the origin of that build, to help with filtering and grouping. E.g. `pr`, `beta`, `final-release`',
```

Reply:
Is there a reason why the example values here are lower case while in the previous item they are capitalized?

Reply:
Not really 😅 

The actual reason is that the examples for the other keys are values already existing and provided by the project configuration (e.g. `Debug` and `Release` are build configuration names spelled capitalized in Xcode's build settings; `Vanilla`, `Jalapeno`, etc (for the `android_send_size_metrics.rb`'s `product_flavor`) are also spelled capitalized in the `./gradlew assembleWordpressVanillaRelease` commands, etc… while this `source` key is designed to be totally arbitrary, independant to the build configurations and settings of the project, and more acting like "hashtags" to attribute to a build metric.

So I guess I thought about the `#hashtag` similarity here and that's why I went with lowercase in the examples… 🤷